### DISCUSSION TITLE
When will Spock 2.2-M2 be released?

### DISCUSSION THREAD

**Discussion by erling** [spockframework/spock#1434] [Q&A] [answered]
When will Spock 2.2-M2 be released? (Bumped into this one https://github.com/spockframework/spock/pull/1407)

Thanks :)

**Comment by leonard84** [comment, marked as answer]
We could probably do a release this week(end) if there is a need for it.

**Reply by erling** [reply]
It would be really valuable to us if there is a release within a couple of weeks. Thanks a lot 😊

**Reply by erling** [reply]
Really hope a 2.2-M2 release is around the corner. Thanks in advance.  @leonard84

**Reply by erling** [reply]
Still hoping for a 2.2-M2 release. Thanks in advance @leonard84

**Reply by vanta** [reply]
@leonard84 It's already June :) Any plans for the next release?

**Reply by leonard84** [reply]
Spock 2.2-M2 and M3 has been released today.
